keps/sig-storage/3141-prevent-volume-mode-conversion/README.md

```diff
@@ -539,18 +593,15 @@ These goals will help you determine what you need to measure (SLIs) in the next
 question.
 -->
 
+N/A.
```

Comment:
It seems reasonable to expect a certain rate of persistentvolumeclaim_provision_failed_total versus a successful count.

Reply:
Being able to count the number of provisioning failures caused by a lack of permissions or invalid usage can be used as an indicator of a systemic problem when compared against a total.  While it will often/usually be a usage error, a simple count here can aid significantly in detection (same as bad node selectors causing pods to not schedule, which we also watch).